src/content/docs/new-relic-solutions/get-started/intro-new-relic2.mdx

```diff
@@ -0,0 +1,515 @@
+  />
+
+  </Side>
+  </SideBySide>
+
+# IAST [#iast]
+<SideBySide>
+  <Side>
+
+  <img
+  title="New Relic IAST"
+  alt="New Relic IAST"
+  src={NRIastMain}
+  />
+
+  </Side>
+  <Side>
+
+  New Relic Interactive Application Security Testing (IAST) can help you stop cyberattacks and breaches on your applications. It does this by probing your running code for exploitable flaws.
+
+  Learn more about IAST [here](/docs/iast/introduction).
+
+  </Side>
+  </SideBySide>
+
+# Infrastructure [#infrastructure]
+<SideBySide>
+  <Side>
+
+  The <InlinePopover type="infrastructure" /> capability lets you monitor the health and performance of your service. It covers containers and cloud infrastructure. It includes tracking CPU, memory, network traffic, and disk use. It also includes over 400 on-host integrations for monitoring third-party apps. 
+
+  Learn more about infrastructure [here](/docs/infrastructure/infrastructure-monitoring/get-started/get-started-infrastructure-monitoring/).
+
+  </Side>
+  <Side>
+
+  <img
+    title="New Relic infrastructure"
+    alt="New Relic infrastructure"
+    src={NRInfraMain}
+  />
+  </Side>
+  </SideBySide>
+
+# Kubernetes and Pixie [#k8s-pixie]
+<SideBySide>
+  <Side>
+
+  <img
+    title="New Relic Kubernetes"
+    alt="New Relic Kubernetes"
+    src={NRK8sMain}
+  />
+
+  </Side>
+  <Side>
+
+  The New Relic Kubernetes integration gives you full observability. The data it provides lets you monitor the health of your whole Kubernetes cluster and the performance of your environment. You can check individual pods or drill into specific services and applications. The agent collects telemetry data from your cluster. It uses several New Relic integrations. These include Kubernetes events, the Prometheus Agent, and the New Relic Logs Kubernetes plugin.
+
+  Learn more about Kubernetes [here](l/docs/kubernetes-pixie/kubernetes-integration/get-started/introduction-kubernetes-integration).
+
+  </Side>
+  </SideBySide>
+
+# Live archives [#live-archives]
+<SideBySide>
+  <Side>
+
+  Live archives store historical logs with other logs and telemetry data in the New Relic Database. This allows instant access to historical logs and eliminates the need to rehydrate, reload, re-index, or move data to many places. 
+
+  Learn more about live archives [here](/docs/logs/get-started/log-live-archives).
+
+  </Side>
+  <Side>
+
+  <img
+    title="New Relic Live archives"
+    alt="New Relic Live archives"
+    src={NRLiveArchivesMain}
+  />
+
+  </Side>
+  </SideBySide>
+
+# Logs [#logs]
+<SideBySide>
+  <Side>
+
+  <img
+    title="New Relic Live archives"
+    alt="New Relic Live archives"
+    src={NRLogsMain}
+  />
+
+  </Side>
+  <Side>
+
+  New Relic can help you manage your large volume of log data. We provide tools to ingest only valuable logs to cut costs. We offer a unified UI to connect your <InlinePopover type="logs" /> to your services. We also offer ways to organize them before you drown in them.
```

Comment:
I think "drown in them" is a little too harsh, I'd reword this and soften it a bit